internal/versions/versions.go

```diff
@@ -338,6 +358,83 @@ func (h *handler) validateVersions() error {
 	return nil
 }
 
+func existAlready(db *gorm.DB, openShiftReleaseImage common.OpenShiftReleaseImage) bool {
+	var count int64
+	db.Model(&common.OpenShiftReleaseImage{}).
+		Where("openshift_version = ? AND channel = ? AND cpu_architecture = ?", openShiftReleaseImage.OpenshiftVersion, openShiftReleaseImage.Channel, openShiftReleaseImage.CpuArchitecture).
+		Count(&count)
+
+	return count > 0
+}
+
+func syncReleaseImages(db *gorm.DB, releaseSources models.ReleaseSources) error {
+	for _, releaseSource := range releaseSources {
+		openshiftVersion := *releaseSource.OpenshiftVersion
+		for _, upgradeChannel := range releaseSource.UpgradeChannels {
+			cpuArchitecture := *upgradeChannel.CPUArchitecture
+			for _, channel := range upgradeChannel.Channels {
+				// OCP releases API needs amd64, arm64 instead of x86_64, aarch64 respectively
+				if cpuArchitecture == "x86_64" {
```

Comment:
using a map instead of if-else will improve readability and improve this translating back-and-forth

```suggestion
cpuArchDict := map[string]string{
  "x86_64": "amd64",
  "aarch64": "arm64",
}
cpuArch := defaultArch
if  arch, ok :=  cpuArchDict[cpuArchitecture]; ok {
   cpuArch := arch
}
```

Reply:
Nice